docs/source/release/v6.11.0/framework.rst

```diff
@@ -10,66 +10,103 @@ Algorithms
 
 New features
 ############
-.. amalgamate:: Framework/Algorithms/New_features
+- Errors due to H5 file exceptions are now caught with improved error reporting.
+- Algorithm :ref:`algm-Load` will now give a warning if the extension specified on the filename, eg. `MUSR15189.txt`, is not found.
+  Loading files from interfaces or other algorithms should also give this warning.
+  The algorithm will still load the file by looking for other extensions, as was the case before.
+- Algorithm :ref:`GenerateGroupingPowder <algm-GenerateGroupingPowder>` now has a newer version 2 that will save the grouping file with groups starting at 1 instead of 0 to make them consisted with ``GroupingWorkspace``.
```

Comment:
```suggestion
- Version 2 of algorithm :ref:`GenerateGroupingPowder <algm-GenerateGroupingPowder>` that will save the grouping file with groups starting at 1 instead of 0 to make them consisted with ``GroupingWorkspace``.
```